app/main/controller/story_controller.py

```diff
@@ -0,0 +1,50 @@
+# endpoint for user operations
+from flask import request, abort
+from flask_login import current_user, login_required
+from flask_restplus import Resource
+
+from app.main.util.dto import StoryDto
+from app.main.service.story_service import StoryService as Story
+
+api = StoryDto.api
+story = StoryDto.story
+queryParams = StoryDto.queryParams
+
+@api.route('/addStory')
+class AddNewStory(Resource):
+    """ Add a new story """
+    @api.doc('Endpoint to add a new story')
+    @api.expect(story, validate=True)
```

Comment:
Adding story should require the user to be logged is as the superuser as we have for other POST methods.